app/src/main/java/com/travel/travelmate/LoginActivity.java

```diff
@@ -84,11 +84,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         // Check if there is userId in "sharedPreferences" object
         // If yes, then directly go to MainActivity
         sharedPreferences = getEncryptedSharedPrefs();
-        if(sharedPreferences == null) {
-            sharedPreferences = getSharedPreferences(Const.SHAREDPREFERENCE, MODE_PRIVATE);
-        }
-        userId = sharedPreferences.getString(UserId, "0");
-        if (!TextUtils.equals(userId, "0")) {
+        if (sharedPreferences.contains(UserId)) {
```

Comment:
No longer needed to check if sharedPreferences exists.
We just nee to check if UserId exists in sharedPreferences.